packages/core/src/domain/tracekit/tracekit.ts

```diff
@@ -55,41 +55,36 @@ export function startUnhandledErrorCollection(callback: UnhandledErrorCallback)
  */
 function instrumentOnError(callback: UnhandledErrorCallback) {
   return instrumentMethodAndCallOriginal(window, 'onerror', {
-    before(this: any, message: Event | string, url?: string, lineNo?: number, columnNo?: number, errorObj?: Error) {
-      let stack: StackTrace
-
+    before(this: any, messageObj: unknown, url?: string, line?: number, column?: number, errorObj?: unknown) {
       if (errorObj) {
-        stack = computeStackTrace(errorObj)
-        callback(stack, errorObj)
-      } else {
-        const location = {
-          url,
-          column: columnNo,
-          line: lineNo,
-        }
-
-        let name
-        let msg = message
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏I think you missed this: if `message` is a string, it should be the default value in case it does not match `ERROR_TYPES_RE`

Reply:
With 
```javascript
const ERROR_TYPES_RE =
  /^(?:[Uu]ncaught (?:exception: )?)?(?:((?:Eval|Internal|Range|Reference|Syntax|Type|URI|)Error): )?(.*)$/
```
if message is a string it should always match, no?

Reply:
Ah that's right, so you could consider that `ERROR_TYPES_RE.exec(messageObj as string)` always succeeds then.